### DISCUSSION TITLE
[Feature Request] Add option to change desired Tweet URL (nitter)

### DISCUSSION THREAD

**Discussion by Still34** [robertodoering/harpy#849] [Ideas]
## Summary

When sharing tweets, it would be nice to have an option to share the tweet as a nitterized link using a hosted Nitter instance (e.g., nitter.net). This should be relatively easy to do as the base URL is defined under `lib/api/twitter/data/tweet_data.dart` - I've already tried it out on my own for my local build of harpy and works perfectly. Ideally, this should be an option under the settings, where the user can choose from a range of known Nitter instances or allow the user to enter their own.

**Comment by robertodoering** [comment]
@Still34
Instead of trying to maintain a list of known Nitter instances maybe just a textfield to set the host would be good enough (with nitter.net as the default)
If you are interested to add this to harpy feel free to work on it and create a PR
